graphs/bellman_ford.py

```diff
@@ -1,56 +1,53 @@
-from __future__ import annotations
+from typing import Dict, List
```

Comment:
This is the opposite of what we want.

Reply:
Do we want to remove type annotation? I think we need to fix mypy annotation, but not to remove.

Reply:
We want to use `dict, list` not `Dict, List`

Reply:
I haven't found this requirement at the issue #4052. Is it mentioned elsewhere?

Reply:
#2464

Reply:
Ah, ok. Sorry , As I am new here, I couldn't notice that. I will keep that in mind for next PRs.